aten/src/ATen/native/native_functions.yaml

```diff
@@ -1829,6 +1830,7 @@
 - func: smm(Tensor self, Tensor mat2) -> Tensor
   variants: function, method
 
+# log_softmax doesn't follow kwargs-only convention because BC-breaking when loading jit models that use this operator.
```

Comment:
s/log_softmax/softmax

Reply:
oops, copy/pasted when I changed the wording. fixed.